### ISSUE TITLE
[high] Keyboard Navigation: Licenses: The state expandable sections are not focusable using keyboard

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5500]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Tab to focus the "licenses" link and press Enter key.
3. Press Tab key and try to focus the state expandable sections.

## Expected Result:
The state expandable sections should be focusable and accessible using keyboard only.

## Actual Result:
The state expandable sections cannot be focused using Tab key.

HTML:
`<div class="toggle-title js_infoToggle"><h4>Alabama <span class="icon is-blue"><i class="expensicons expensicons-xs expensicons-caret-down"></i></span></h4></div>`

## Workaround:
No workaround

## Area issue was found in:
Licenses page

## Failed WCAG checkpoints
2.1.1

## User impact:
The state expandable sections cannot be focused or operated by people who use keyboard only.

## Suggested resolution:
Use native <button> to implement the state expandable sections. See an example of implementation here: https://accessibility.huit.harvard.edu/technique-expandable-sections

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134719809-ff2c9725-7a3d-4bcc-8d52-ecc3ff3fe6f2.mp4

![Bug5181344_Keyboard_for_dropdowns-Lincenses](https://user-images.githubusercontent.com/88733897/134719818-41968ea9-99dd-4f98-a78b-09304cde0749.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
Same comment: So the terms of service page is hosted on UseDot and it's technically not part of the NewDot app. @stitesExpensify @kadiealexander do you still think it's worth addressing these issues, or should we punt and focus on the app itself?
